x/dclupgrade/client/cli/tx_approve_upgrade.go

```diff
@@ -0,0 +1,42 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/utils/cli"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/x/dclupgrade/types"
+)
+
+func CmdApproveUpgrade() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "approve-upgrade [name]",
+		Short: "Approve proposed upgrade with given name",
+		Args:  cobra.ExactArgs(1),
```

Comment:
We usually use named flags in other commands, not arguments. Probably let's make it as in other modules (--name flag)?

Reply:
Done.